src/OpenTelemetry.Instrumentation.AspNet/Implementation/HttpInListener.cs

```diff
@@ -57,12 +57,7 @@ public void Dispose()
         /// <returns>Span uri value.</returns>
         private static string GetUriTagValueFromRequestUri(Uri uri)
         {
-            if (string.IsNullOrEmpty(uri.UserInfo))
-            {
-                return uri.ToString();
-            }
-
-            return string.Concat(uri.Scheme, Uri.SchemeDelimiter, uri.Authority, uri.PathAndQuery, uri.Fragment);
+            return string.IsNullOrEmpty(uri.UserInfo) ? uri.ToString() : string.Concat(uri.Scheme, Uri.SchemeDelimiter, uri.Authority, uri.PathAndQuery, uri.Fragment);
```

Comment:
Would this line be too long? (e.g. the original version makes it easier to put a breakpoint on Ln62).